auth-web/src/util/constants.ts

```diff
@@ -435,3 +435,10 @@ export enum RequestTrackerType {
     ChangeStatus = 'CHANGE_STATUS',
     ChangeParty = 'CHANGE_PARTY',
 }
+
+export enum EntityDetailTypes {
```

Comment:
I would rename this EntityAlertType or something like that so its more clear what this is for

Reply:
Yup renamed